### DISCUSSION TITLE
Please Add a Goal Name

### DISCUSSION THREAD

**Discussion by Chaiavi** [plausible/analytics#838] [Anything Else]
When adding a goal, I add only the page path
I want another field with the goal name, so I will understand easily in the dashboard the goal without trying to parse the path in my mind in order to understand what this goal does.

For example, if I have a goal for getting to the checkout page, I want an option to add a name to this goal, something like: "Checkout Page", so when I see my dashboard with the goal numbers, I will understand what the URL means easily

This seems like a really small thing but has a nice impact - Please upvote

**Comment by Spone** [comment]
Totally agree, this would allow us to give meaningful Goal names, such as "Sent contact form" instead of:

![image](https://user-images.githubusercontent.com/38524/226894222-ba0044d3-e7af-415b-a264-3309795b4050.png)
